src/client.ts

```diff
@@ -259,11 +260,35 @@ bad input into one of the iframe-coordinator client methods.
     this._isStarted = true;
 
     this._clientWindow.addEventListener('message', this._onWindowMessage);
-    this._clientWindow.addEventListener('click', this._onWindowClick);
```

Comment:
Why not leave the default behavior intact and provide an option to start for deciding wether or not to intercept the links?

Then we don't have to bump the major.

Reply:
We could, but IMO the separate API call is nicer and it's pretty easy to do the major bump now while the library isn't widely used. Also, since it doesn't introduce client/host compatibility issues, it won't force anybody having success with an older version to upgrade.  That said, I'm open to the idea if I've missed a reason this will cause pain for somebody.

Reply:
Agreed, I'd rather do the major bump now. Happy to change though if needed.